### PR TITLE
Feature: AMQP Messages

### PR DESCRIPTION
**Pull request by jlamarr22**
Changes to support AMQP Provider 

Other PRs:
- https://github.com/EOSIO/eosio-java-abieos-serialization-provider/pull/2
- https://github.com/EOSIO/eosio-java-android-abieos-serialization-provider/pull/55
- https://github.com/EOSIO/eosio-java-rpc-provider/pull/3
- https://github.com/EOSIO/eosio-java-android-rpc-provider/pull/51
- https://github.com/EOSIO/eosio-java-amqp-provider (new provider)
- https://github.com/EOSIO/eosio-java-rabbitmq-integration (proof of concept)

### CODE REVIEW

File: eosiojava/build.gradle [hunk 1]
```diff
@@ -36,12 +36,12 @@ dependencies {
...
     testCompile 'org.mockito:mockito-core:2.26.0'
     api 'org.bitcoinj:bitcoinj-core:0.15.2'
-    api 'org.slf4j:slf4j-jdk14:1.7.25'
+    implementation 'io.reactivex.rxjava2:rxjava:2.2.19'
 }
 
 def libraryGroupId = 'one.block'
 def libraryArtifactId = 'eosiojava'
-def libraryVersion = '0.1.2'
+def libraryVersion = '0.2.0-amqp9'
```
Comment (wynvandevb1) [inline eosiojava/build.gradle:45]: what version do we actually want to bump this to?

Comment (opi-smccoole) [reply eosiojava/build.gradle:45]: Undetermined at this point.  Currently the next release is targeted toward eosio-2.1 features and amqp support is not scheduled to be part of that release.  If there is a need for it @brandonfancher should be consulted to put it into the release plan.

Comment (wynvandevb1) [reply eosiojava/build.gradle:45]: It is needed to support our AMQP provider/plug-in, and we plan to use that on services in the near future so would like to merge this in @brandonfancher @opi-smccoole

**Comment by opi-smccoole** [comment]
The packed transaction support part of this was merged as #121.  The rest was migrated to the AMPQ provider repo as it had heavy dependencies (RxJava) and really was not generally applicable to most users.

**Pull request closed without merging**